Source/Text/SvgTextBase.cs

```diff
@@ -19,10 +19,19 @@ public abstract partial class SvgTextBase : SvgVisualElement
 
         public SvgTextBase()
         {
-            _x.CollectionChanged += OnXChanged;
-            _dx.CollectionChanged += OnDxChanged;
-            _y.CollectionChanged += OnYChanged;
-            _dy.CollectionChanged += OnDyChanged;
+            NotifyCollectionChangedEventHandler xHandler = null;

```

Comment:
It is enough to set `Attributes` only once, which is changed from default value.